google-compute-engine/src/test/java/org/jclouds/googlecomputeengine/features/ProjectApiLiveTest.java

```diff
@@ -66,4 +67,14 @@ public void testDeleteItemFromMetadata() {
       assertEquals(project.commonInstanceMetadata().size(), initialMetadataSize);
       assertEquals(project.commonInstanceMetadata().fingerprint(), initialFingerprint);
    }
+
+   @Test(groups = "live", dependsOnMethods = "getProject")
+   public void testSetUsageExportBucket() {
+      Operation o = api.project().setUsageExportBucket("test-bucket", "test-");
+      while (o.error().errors().isEmpty()) {
+         o = api.operations().get(o.selfLink());
+      }
```

Comment:
Just to make sure I understand this test. Is its purpose to test that operation errors are reported properly? In other words, can we properly live test the `setUsageExportBucket` without having that loop?